app/src/main/AndroidManifest.xml

```diff
@@ -14,15 +15,15 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.KloBuddy"
-        tools:targetApi="31" >
+        tools:targetApi="31">
 
         <meta-data
             android:name="com.google.android.geo.API_KEY"
             android:value="${MAPS_API_KEY}" />
 
         <activity
             android:name=".MainActivity"
-            android:exported="true" >
```

Comment:
Do you want to feel special by deleting whitespace?
![images](https://github.com/julix14/wc-buddy/assets/65715000/b7952fae-ca06-4bc5-a501-4f5d272d8411)